hazelcast/src/main/java/com/hazelcast/spi/NodeEngine.java

```diff
@@ -287,4 +288,6 @@
      * @since 3.8
      */
     MemberVersion getVersion();
+
+    ClusterWideConfigurationService getConfigurationService();
```

Comment:
Also `ConfigurationService ` is in `com.hazelcast.internal.dynamicconfig` package which is internal.